config/config.go

```diff
@@ -39,6 +40,8 @@ type Flags struct {
 	Help            bool
 	Provider        string
 	SkipV2          bool
+	PushGatewayURL  string
+	PushGatewayJob  string
```

Comment:
in most cases, people tend to put variables in alphabetical order, its good to get used to it :)

Reply:
sorted all structs alphabetically